### DISCUSSION TITLE
Is there a way to 'resize' grain data without reconstructing them via calcGrains?

### DISCUSSION THREAD

**Discussion by Remsdel** [mtex-toolbox/mtex#1768] [Ask Anything]
Here is the following straightforward issue:
I am faced with a large numbers of processed EBSD maps for which the scale was incorrectly calibrated. After a new calibration, a correction factor (CF) for x and y scales was determined.
It is trivial to rescale the ebsd data via the following commands:
ebsd.x=ebsd.x*CF;
ebsd.y=ebsd.y*CF;
Ultimately I am interested in the data for grain areas but I would like to avoid re-running a grain reconstruction for several reasons but mostly because of the sheer amount of data for which I would need to run the calcgrains function, which would take a lot of time (many larges map with denoising steps). Is there another and simpler way to rescale grain 'sizes' similar to the ebsd.x=ebsd.x*CF without a reconstruction step?
Thanks in advance

**Comment by kilir** [comment]
Hi,
how about that? Just check if there is somethign missing;

```
mtexdata titanium
grains  = ebsd.calcGrains
%
plot(grains,'coordinates','on')
nextAxis
fac1 = 0.8;
fac2 = 2

% for ebsd
ebsd.x = ebsd.x * fac1;
ebsd.y = ebsd.y * fac2;

% for grains
cf = repmat([fac1 fac2],length(grains.V),1)
grains.V = grains.V .* cf

plot(ebsd,'coordinates','on')
hold on
plot(grains.boundary)
hold off

```
Cheers,
Rüdiger

**Comment by Remsdel** [comment]
That does the job, fantastic,
thank you
Cheers
Rémi
